gitgud/skills/intro/__init__.py

```diff
@@ -73,13 +76,14 @@ def status(self):
         else:
             self.display_message("status-norepo.txt")
```

Comment:
`git gud status` doesn't produce correct output. It always thinks we're in a repo